message/src/main/java/com/example/message/controller/MessageController.java

```diff
@@ -5,23 +5,20 @@
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
 
-import java.net.InetAddress;
-import java.net.UnknownHostException;
-
 @RestController
 public class MessageController {
-    @GetMapping(value = "/message")
+    @GetMapping(value = "/getMessage")
     public ResponseEntity<String> message() {
-        String hostname = getHostname();
+        String hostname = "Message from host";
         return new ResponseEntity<>(hostname, HttpStatus.OK);
     }
 
-    private String getHostname() {
+    /*private String getHostname() {
```

Comment:
Why did you comment this?

Reply:
This method was commented initially for testing purpose. Will remove the comment from this method.